ironfish-cli/src/commands/wallet/chainport/send.ts

```diff
@@ -0,0 +1,421 @@
+    const { flags } = await this.parse(BridgeCommand)
+
+    const networkId = (await client.chain.getNetworkInfo()).content.networkId
+
+    if (networkId !== TESTNET.id) {
+      this.error(`Chainport transactions are only available on testnet.`)
+    }
+
+    if (!flags.offline) {
+      const status = await client.wallet.getNodeStatus()
+
+      if (!status.content.blockchain.synced) {
+        this.error(
+          `Your node must be synced with the Iron Fish network to send a transaction. Please try again later`,
+        )
+      }
+    }
+
+    const { targetNetwork, from, to, amount, asset, assetData } =
+      await this.getAndValidateInputs(client, networkId)
+
+    const rawTransaction = await this.constructBridgeTransaction(
+      client,
+      networkId,
+      targetNetwork,
+      from,
+      to,
+      amount,
+      asset,
+      assetData,
+    )
+
+    await confirmOperation({})
+
+    const postTransaction = await client.wallet.postTransaction({
+      transaction: RawTransactionSerde.serialize(rawTransaction).toString('hex'),
+      account: from,
+    })
+
+    const bytes = Buffer.from(postTransaction.content.transaction, 'hex')
+    const transaction = new Transaction(bytes)
+    const hash = transaction.hash().toString('hex')
+
+    if (postTransaction.content.accepted === false) {
+      this.warn(`Transaction '${hash}' was not accepted into the mempool`)
+    }
+
+    const transactionUrl = getExplorer(networkId)?.getTransactionUrl(hash)
+
+    if (transactionUrl) {
+      this.log(`\nBlock explorer: ${transactionUrl}`)
+    }
+
+    if (flags.watch) {
+      this.log('')
+
+      await watchTransaction({
+        client,
+        logger: this.logger,
+        account: from,
+        hash,
+      })
+    } else {
+      this.log(`Run ironfish wallet:transaction ${hash} to get the status of your transaction.`)
+      this.log(
+        `Run ironfish wallet:transaction:watch ${hash} to wait for the status of your transaction to be confirmed/ expired.`,
+      )
+    }
+  }
+
+  private async getAndValidateInputs(client: RpcClient, networkId: number) {
+    const { flags } = await this.parse(BridgeCommand)
+
+    let from = flags.account?.trim()
+    let to = flags.to?.trim()
+    let assetId = flags.assetId
+
+    if (!from) {
+      const response = await client.wallet.getDefaultAccount()
+
+      if (!response.content.account) {
+        this.error(
+          `No account is currently active.
+            Use ironfish wallet:create <name> to first create an account`,
+        )
+      }
+
+      from = response.content.account.name
+    }
+
+    if (!to) {
+      to = await CliUx.ux.prompt('Enter the public address of the recipient', {
+        required: true,
+      })
+    }
+
+    if (!validator.isAddress(to)) {
+      this.error('Invalid to ethereum address')
+    }
+
+    if (flags.expiration !== undefined && flags.expiration < 0) {
+      this.error('Expiration sequence must be non-negative')
+    }
```

Comment:
We added `promptExpiration` last week, could be useful here

Reply:
I think to use this prompt, we would have to add support for the `--rawTransaction` flag. Should we add this now?

Reply:
Yeah I think that it makes more sense to do it when there's support for `rawTransaction` since this will just use the default expiration delta. I don't think we need to add support for `rawTransaction` before this goes in